test/e2e/framework/helper/helper.go

```diff
@@ -17,10 +17,10 @@ limitations under the License.
 package helper
 
 import (
+	cmclient "github.com/cert-manager/cert-manager/pkg/client/clientset/versioned"
 	"k8s.io/client-go/kubernetes"
 
 	"github.com/cert-manager/cert-manager/e2e-tests/framework/config"
```

Comment:
nb: I think this is probably the most controversial aspect of the change. ATM the GCI config treats all cert-manager modules in this repo as being similar for import purposes, and this change makes the distinction between the modules much clearer.

I'm quite relaxed about it either way, though. This PR has some other tweaks which look aesthetically better, and mostly I'm pretty neutral on this overall because it doesn't affect functionality.

On that basis, LGTM but maybe others will disagree!

/lgtm